src/pages/roadmap/index.tsx

```diff
@@ -18,86 +18,265 @@ type PeriodData = {
+              but it&apos;s problematic for example in the Arabic language,
+              where it&apos;s hard to mix HTML and Arabic characters.
+            </p>
+            <p>
+              By visual editor, we mean a nice environment, where tags and
+              parameter placeholders are represented visually, so translators
+              don&apos;t have to worry about them. The plurals are also visually
+              represented, so users can edit them without understanding the
+              message formatting language.
+            </p>
+          </>
+        ),
       },
       {
-        name: 'Support for mobile apps',
-        description: `Tolgee works great with Web apps, and we would like to provide the same 
-comfort to mobile developers.`,
+        name: 'Translator Marketplace',
+        description: (
+          <p>
+            Looking for a translator to access foreign markets? Tolgee will help
+            you with this. With the Tolgee Translator Marketplace, there will be
+            nothing easier than finding the ideal translator for your app.
+          </p>
+        ),
       },
       {
-        name: 'QA Checks',
-        description: `Even translators make mistakes, but some of these mistakes can be detected, 
-and translators can be notified. No more mistakes like missing tags, missing 
-parameters, or missing punctuation marks.`,
+        name: 'Branching',
+        description: (
+          <p>
+            Sometimes you need to maintain multiple versions of your software,
+            so you’ll need to have multiple versions of your localization data
+            stored in Tolgee as well.
+          </p>
+        ),
       },
       {
         name: 'Git integrations',
-        description: [
-          `There are times when it would be great if localization data were synced with 
-your Git repository. Or maybe you would like to create pull requests 
-automatically when translations are changed...`,
-          `To do this, integrations to <b>GitHub, GitLab</b>, or just support for Git itself will be 
-helpful.`,
-        ],
+        description: (
+          <>
+            <p>
+              There are times when it would be great if localization data were
+              synced with your Git repository. Or maybe you would like to create
+              pull requests automatically when translations are changed...
+            </p>
+            <p>
+              To do this, integrations to <b>GitHub, GitLab</b>, or just support
+              for Git itself will be helpful.
+            </p>
+          </>
+        ),
       },
       {
-        name: 'Branching',
-        description: `Sometimes you need to maintain multiple versions of your software, so you’ll 
-need to have multiple versions of your localization data stored in Tolgee as 
-well.`,
+        name: 'Backups and data export',
+        description: (
+          <>
```

Comment:
Use p tag instead of <>. There is no padding between items.
![Snímek obrazovky 2022-09-29 v 12 25 20](https://user-images.githubusercontent.com/8831144/193010254-f1dcc20b-1bab-4661-a93c-701718f324e9.png)